### PR TITLE
OSSMDOC-717: disabling auto injection does not work

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OSSMDOC-717

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-717/service_mesh/v2x/ossm-vs-community.html

QE review:
- [x] QE has approved this change.

Additional information:

SME review: Jay S
QE review: Matt M
Peer review: Brian B

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@jshaughn I've made this doc change based on your comment in OSSMDOC-717. Please review when you have time. Thanks!

**Review by jshaughn — APPROVED** [review]
LG

**Comment by mattmahoneyrh** [comment]
LGTM

File: modules/ossm-vs-istio.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ The command line tool for {SMProductName} is `oc`.  {SMProductName} does not s
 
 The upstream Istio community installation automatically injects the sidecar into pods within the projects you have labeled.
 
-{SMProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` annotation as described in the Automatic sidecar injection section.
+{SMProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` label, or annotation, as described in the Automatic sidecar injection section.
```
Comment (sounix000) [inline modules/ossm-vs-istio.adoc:30]: Suggestions (mostly subjective nitpicks):
- `but requires you to opt in to injection` --> "but you must opt in to injection"
- `To enable automatic injection you specify` --> "To enable automatic injection, specify"
- `described in the Automatic sidecar injection section` --> Although I am aware about the OpenShift xref guidelines, from a reader's perspective, a link to that section would be nice.

As a whole, LGTM.

Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc:30]: Thanks for the style suggestion, @sounix000 I've made the updates; however, I am not able to link across repos.

Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc]: Made style updates; unable to link across repos.

File: modules/ossm-vs-istio.adoc [hunk 2]
```diff
@@ -27,7 +27,7 @@ The command line tool for {SMProductName} is `oc`.  {SMProductName} does not s
 
 The upstream Istio community installation automatically injects the sidecar into pods within the projects you have labeled.
 
-{SMProductName} does not automatically inject the sidecar to any pods, but requires you to opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection you specify the `sidecar.istio.io/inject` annotation as described in the Automatic sidecar injection section.
+{SMProductName} does not automatically inject the sidecar to any pods, but you must opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other OpenShift capabilities such as builder pods. To enable automatic injection, specify the `sidecar.istio.io/inject` label, or annotation, as described in the Automatic sidecar injection section.
```
Comment (bburt-rh) [inline modules/ossm-vs-istio.adoc:30]:
```suggestion
{SMProductName} does not automatically inject the sidecar into any pods, but you must opt in to injection using an annotation without labeling projects. This method requires fewer privileges and does not conflict with other {product-title} capabilities such as builder pods. To enable automatic injection, specify the `sidecar.istio.io/inject` label, or annotation, as described in the _Automatic sidecar injection_ section.
```

Comment (rh-tokeefe) [reply modules/ossm-vs-istio.adoc:30]: PR updated

**Pull request merged**